content/en/docs/tasks/observability/metrics/classify-metrics/index.md

```diff
@@ -0,0 +1,325 @@
+---
+title: Classifying Metrics Based on Request or Response (Experimental)
+description: This task shows you how to improve telemetry by grouping requests and responses by their type. 
+weight: 27
+keywords: [telemetry,metrics,classify,request-based]
+---
+
+It is useful to visualize telemetry based on the type of requests and responses
+handled by services in your mesh. For example, a bookseller tracks the number of
+times their web site gets book reviews from a backend service. A book review
+request has this structure:
+
+{{< text plain >}}
+GET /reviews/{review_id}
+{{< /text >}}
+
+Counting the number of review requests becomes complicated by the
+unbounded element, `review_id`. If you use this metric as-is, it would create a
+dimension for every unique `review_id` and obfuscate the meaning of the
+results. To resolve this problem, Istio lets you create classification rules
+that group requests into a more useful dimension for your telemetry, such as
+creating and counting the results of a higher-level `GetReviews` dimension.
+
+The classification rules follow the syntax used by
```

Comment:
Remove references to `HTTPRouteMatch`

Reply:
The API is defined here now https://preliminary.istio.io/docs/reference/config/proxy_extensions/attributegen

however it is not available there yet...

Reply:
Done